README.md

```diff
@@ -30,13 +30,13 @@ The current members of the Presto TSC are:
 
 | Name | Github | Term begins | Term ends | Affiliation |
 | ---- | ------ | ------------|-------------|-----------|
-| Bin Fan  | apc99 | November 1, 2021 | November 1, 2022 | Alluxio |
-| James Sun  | highker | November 1, 2021 | November 1, 2022 | Facebook |
+| James Sun  | highker | January 23, 2023 | November 1, 2022 | Meta |
+| Deepak Majeti  | majetideepak | January 10, 2023 | November 1, 2023 | Ahana Cloud |
 | Rebecca Schlussel | rschlussel | November 1, 2021 | May 1, 2023 | Facebook |
-| Rongrong Zhong | rongrong | November 1, 2021 | May 1, 2023 | Facebook |
-| Ying Su  | yingsu00 | November 1, 2021 | November 1, 2022 | Ahana Cloud |
-| Zhenxiao Luo  | zhenxiao | November 1, 2021 | May 1, 2023 | Twitter |
-| Tim Meehan  | tdcmeehan | November 1, 2021 | November 1, 2022 | Facebook |
+| Rongrong Zhong | rongrong | November 1, 2021 | May 1, 2023 | Meta |
```

Comment:
should we update Rongrong's affiliation?

Reply:
Done!